### DISCUSSION TITLE
Can't make it (desktop app) work with a Google Cloud project via OAuth 2.0

### DISCUSSION THREAD

**Discussion by fredericsimard** [hoppscotch/hoppscotch#4346] [General]
Hi! I started evaluating Hoppscotch as a replacement for Postman for our team as we want to use the shared workspaces but Postman asked for 200$/m with other features we don't need.

I'm using Hoppscotch macOS desktop client, not the Progressive web app or the cloud version.

So we use Google Cloud a lot, and I'm trying to setup a small test project I already used with Postman just fine, but for some reason Google rejects Hoppscotch when I try to generate the token, with error:
```
Authorization Error: Error 400: invalid_request
```
and then adds this at the bottom:
```
If you’re the app developer, make sure that these request details comply with Google policies.
• redirect_uri: tauri://localhost/oauth
• flowName: GeneralOAuthFlow
```
however, trying to add `tauri://localhost/oauth` to the authorized credentials generates an error probably because it doesn't start with http. Tried adding `http://localhost/oauth` but still doesn't work.

Any idea why?

**Comment by trymeouteh** [comment]
I am not sure if Hoppscotch desktop app is fully open source. I asked on Github months ago and never got a detailed response.

**Reply by liyasthomas** [reply]
Source code for desktop app: https://github.com/hoppscotch/hoppscotch/tree/main/packages/hoppscotch-selfhost-desktop
